### PR TITLE
IRONSIDE-1156 -- merge release/connector/1.3.0 to develop

### PR DESCRIPTION
**Pull request by alvinkrebs**
Per the 7.3.0 release requirements, merge the 1.3.0 release to develop.
After this is approved merge develop to master.

There was a ci_config.yaml conflict.
The rticonnextdds-connector submodule pointer has changed to the release/connector/1.3.0 branch.

### CODE REVIEW

**Review by Alxe — CHANGES_REQUESTED** [review]
The [`rticonnextdds-connector/config.yaml`](https://github.com/rticommunity/rticonnextdds-connector/compare/6886e8aef0a458e6a6c6ff76bf07d378ceb57b0c...4f9e76dcd870d56ec0f31c1b60ad22cda1a0a235#diff-d8d0422389f03d783e32e627250fe29834bd09c6361640d1ff00661dd6820034) file needs update as well, version should stay as `latest`.

File: rticonnextdds-connector
Comment (Alxe) [inline rticonnextdds-connector:1]: `config.yaml` needs to have `latest` as version.

**Review by Alxe — CHANGES_REQUESTED** [review]

File: docs/release_notes.rst [hunk 1]
```diff
@@ -231,10 +245,19 @@ that Connext DDS features such as Monitoring and Security Plugins are now suppor
 Refer to :ref:`Loading Connext DDS Add-On Libraries` for more information.
 
 What's Fixed in 1.1.0
-""""""""""""""""""""""
+^^^^^^^^^^^^^^^^^^^^^
```
[the remark this answers is not in the document]
Comment (rkorte) [reply docs/release_notes.rst:248]: I think the heading is correct, but the text underneath it is weird: "1.0.0 is the first official release of RTI Connector for JavaScript ..." 

@Alxe, do we need to create a new section at the very bottom of this file called Version 1.0.0, with this introductory text?

**Review by Alxe — APPROVED** [review]

**Review by rkorte — APPROVED** [review]

File: docs/release_notes.rst [hunk 2]
```diff
@@ -291,8 +305,15 @@ This problem has been resolved.
...
 
 
+Vulnerability Assessments
+=========================
+Internally, *Connector* relies on Lua. RTI has assessed the current version of 
+Lua used by *Connector*, version 5.2, and found that *Connector* is not currently 
+affected by any of the publicly disclosed vulnerabilities in Lua 5.2.
+
+
 Version 1.0.0
```
Comment (rkorte) [inline docs/release_notes.rst:315]: This still isn't in the right place. See the output: https://jenkins-community.rti.com/job/ci/job/connector-js/job/rticonnextdds-connector-js-doc/view/change-requests/job/PR-202/Connector_20Documentation/

It should be section 8.3.6.

**Review by rkorte — CHANGES_REQUESTED** [review]

**Review by rkorte — APPROVED** [review]

**Pull request merged**